### PR TITLE
Create SUNAT closes #451

### LINKED ISSUE #451: PROPOSAL: Superintendencia Nacional de Aduanas y de Administración Tributaria (Perú) - SUNAT
## Proposal

**What open data standard are you working with?**
OCDS

**The name of the list (or the organization that manages the list)**
Superintendencia Nacional de Aduanas y de Administración Tributaria - SUNAT

**A suggested code for the list (optional)**
PE-SUNAT

**A short description of the list**
The SUNAT according to its creation Law No. 24829, is a specialized technical agency, attached to the Ministry of Economy and Finance. Its main purpose is to administer the taxes of the national government and the tax and non-tax concepts entrusted to it by Law or according to the inter-institutional agreements entered into, providing the resources required for fiscal solvency and macroeconomic stability; ensuring the correct application of the regulations governing the matter and combating tax and customs crimes in accordance with its powers.

**The country or countries that the list covers**
Perú

**The legal form or organizations that the list covers**
All organizations that have RUC

**A URL for information on the list, and a URL for looking up identifiers (if available)**
Web site  https://www.sunat.gob.pe/
RUC search https://e-consultaruc.sunat.gob.pe/cl-ti-itmrconsruc/jcrS00Alias

**One or more examples of identifier from this list, and how you found them**
20131380951 - MUNICIPALIDAD METROPOLITANA DE LIMA

### PR DESCRIPTION
**Pull request by mrshll1001**
Based on #451

### CODE REVIEW

**Review by kd-ods — CHANGES_REQUESTED** [review]
@mrshll1001, there are a couple of field-level changes that need to be made. (See specific comments). At a higher level: I think this list should be re-focused on the RUC number. By that, I mean, that the code should be pe-ruc and SUNAT should be referred to in the description as the body that curates the list. (Tax authorities can often end up administering several types of tax number, so refocusing like this future-proofs the entry a little.)

Also - the name of the file needs to be adjusted. It should be pe-xxxx.json

Give me a shout if you want to talk through any of the above.

**Comment by kd-ods** [comment]
Hi @mrshll1001 - I reviewed this PR a while back. Let me know if you need any more assistance getting it over the line.

File: lists/pe/RUC.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": {
+    "en": "National Superintendency of Customs and Tax Administration (Peru)",
```
Comment (kd-ods) [inline lists/pe/RUC.json:3]: I think we need:

"en": "RUC Number (Peru)"
"local": "Registro Único del Contribuyente (Perú)"

File: lists/pe/RUC.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "local": "Superintendencia Nacional de Aduanas y de Administración Tributaria (Perú)"
+  },
+  "url": "https://www.sunat.gob.pe/",
+  "description": {
+    "en": "The 'Registro Único del Contribuyente' (RUC) number is an 11-digit tax ID number for any natural or legal person (such as a company) in Peru, and is provided and maintained by SUNAT. The SUNAT, according to its creation Law No. 24829, is a specialized technical agency, attached to the Ministry of Economy and Finance. Its main purpose is to administer the taxes of the national government and the tax and non-tax concepts entrusted to it by Law or according to the inter-institutional agreements entered into, providing the resources required for fiscal solvency and macroeconomic stability; ensuring the correct application of the regulations governing the matter and combating tax and customs crimes in accordance with its powers."
```
Comment (kd-ods) [inline lists/pe/RUC.json:8]: If we remove 'Superintendencia Nacional de Aduanas y de Administración Tributaria' from the name of this list, then it should be added to the description here the first time 'SUNAT' is mentioned.

File: lists/pe/RUC.json [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  },
+  "coverage": [
+    "PE"
+  ],
+  "structure": [
+    "government_agency",
+    "company",
+    "charity"
+  ],
+  "code": "PE-SUNAT",
```
Comment (kd-ods) [inline lists/pe/RUC.json:18]: The code should be changed to PE-RUC.

Also, the name of the file needs to be changed to pe-ruc.json.

**Review by kd-ods — CHANGES_REQUESTED** [review]
A couple of final things needed, @mrshll1001.

File: lists/pe/SUNAT [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  "coverage": [
+    "PE"
+  ],
+  "structure": [
+    "government_agency",
+    "company",
+    "charity"
+  ],
+  "code": "PE-SUNAT",
+  "confirmed": false,
```
[the remark this answers is not in the document]
Comment (mrshll1001) [reply lists/pe/SUNAT]: Done

**Review by kd-ods — APPROVED** [review]
Looking good. I'll merge and deploy.

**Pull request merged**